apps/templates/email/email.html

```diff
@@ -0,0 +1,19 @@
+<html>
+<body style="margin: 0; padding: 0; box-sizing: border-box; font-family: Arial, Helvetica, sans-serif;">
+<div style="width: 100%; background: #efefef; border-radius: 10px; padding: 10px;">
+  <div style="margin: 0 auto; width: 90%; text-align: center;">
+    <h1 style="background-color: rgba(0, 53, 102, 1); padding: 5px 10px; border-radius: 5px; color: white;">{{ title }}</h1>
```

Comment:
what is title?

Reply:
The ticket task was to investigate and propose a solution to send email. There was no demands to create any template. Some changing will be made in 133 ticket with this template, showing the way to customize the appearence.

Reply:
To check the functionality there is a need to insert created function into some handler function and run that handler, with title and name stated in body.

Reply:
updated template. Actually its a 133 ticket with it's functionality